### DISCUSSION TITLE
Correct resize for aspect ratio?

### DISCUSSION THREAD

**Discussion by wes-kay** [ImageMagick/ImageMagick#5675] [Help]
I have an image that I want to compress, unfortunately what I was given has `1664x2432` what would be the proper way to get this into a proper aspect in powers of two or divisible by 4?

**Comment by fmw42** [comment]
What is your IM version and platform?  What do you mean by "compress".  I assume you really mean minimize or make smaller in dimensions, not compression quality.

Assume IM 7,

`magick image.suffix -resize "%[fx:floor(logtwo(1664))]x%[fx:floor(logtwo(2432))]" result.suffix`

will give the closest (smaller) power of 2.
